### PR TITLE
chore: add executors to ftm, add deployer and dripper

### PR DESCRIPTION
**Pull request by jorijnsmit**

### CODE REVIEW

File: helpers/addresses.py [hunk 1]
```diff
@@ -716,7 +720,14 @@
         "ops_deployer": "0xDA25ee226E534d868f0Dd8a459536b03fEE9079b",
         "ops_deployer2": "0xeE8b29AA52dD5fF2559da2C50b1887ADee257556",
         "ops_deployer6": "0x96AC69183216074dd8CFA7A380e873380445EaDc",
+        "ops_executor1": "0xcf4fF1e03830D692F52EB094c52A5A6A2181Ab3F",
+        "ops_executor2": "0x8938bf50d1a3736bdA413510688834540858dAEA",
+        "ops_executor3": "0xC69Fb085481bC8C4bfF99B924076656305D9a25D",
+        "ops_executor4": "0xBB2281cA5B4d07263112604D1F182AD0Ab26a252",
+        "ops_executor5": "0xcDAb3AcC1AD3870a93BB72377092B67e290D76f3",
```
Comment (mitche50) [inline helpers/addresses.py:727]: Can't find this address on any of the ftm multisigs

Comment (jorijnsmit) [reply helpers/addresses.py:727]: you are right, that is a fin ops wallet

File: helpers/addresses.py [hunk 2]
```diff
@@ -716,7 +720,13 @@
         "ops_deployer": "0xDA25ee226E534d868f0Dd8a459536b03fEE9079b",
         "ops_deployer2": "0xeE8b29AA52dD5fF2559da2C50b1887ADee257556",
         "ops_deployer6": "0x96AC69183216074dd8CFA7A380e873380445EaDc",
+        "ops_executor1": "0xcf4fF1e03830D692F52EB094c52A5A6A2181Ab3F",
+        "ops_executor2": "0x8938bf50d1a3736bdA413510688834540858dAEA",
+        "ops_executor3": "0xC69Fb085481bC8C4bfF99B924076656305D9a25D",
+        "ops_executor4": "0xBB2281cA5B4d07263112604D1F182AD0Ab26a252",
         "ops_executor6": "0x66496eBB9d848C6A8F19612a6Dd10E09954532D0",
```
Comment (mitche50) [inline helpers/addresses.py:727]: nit but not sure if you want to keep this clean, you go from executor 4 to 6 after the delete.

Comment (jorijnsmit) [reply helpers/addresses.py:727]: this is to stay consistent with mainnet/other chains's labels.

**Review by mitche50 — APPROVED** [review]

File: helpers/addresses.py [hunk 3]
```diff
@@ -717,7 +721,13 @@
...
+        "ops_executor7": "0xaaE2051c2f74920C6662EF5C9B0d602C40D36DF4",
+        "ops_executor8": "0x10C7a2ca16116E5C998Bfa3BC9CEF464475B18ff",
```
Comment (petrovska-petro) [inline helpers/addresses.py:730]:
```suggestion
        "ops_executor8": "0x0a9af7FAba0d5DF7A8C881e1B9cd679ee07Af8A2",
```

**Review by petrovska-petro — APPROVED** [review]

**Review by jalbrekt85 — APPROVED** [review]
verifying addresses against ether/ftmscan lgtm

**Pull request merged**